### PR TITLE
update journal model with new fields

### PR DESCRIPTION
**Pull request by bill-filler**

### CODE REVIEW

**Review by bessiesteinberg — APPROVED** [review]

File: course_discovery/apps/journal/models.py [hunk 1]
```diff
@@ -23,16 +23,24 @@ class Journal(TimeStampedModel):
         verbose_name=_('UUID'),
     )
     partner = models.ForeignKey(Partner)
+    organization = models.ForeignKey(Organization, default=1)
```
Comment (tuchfarber) [inline course_discovery/apps/journal/models.py]: Why do we want it to default to the first org?

File: course_discovery/apps/journal/models.py [hunk 2]
```diff
@@ -23,16 +23,24 @@ class Journal(TimeStampedModel):
...
     )
+
+    # ecommerce related
     price = models.DecimalField(**PRICE_FIELD_CONFIG)
     currency = models.ForeignKey(Currency)
     sku = models.CharField(max_length=128, null=True, blank=True)
-    expires = models.DateTimeField(null=True, blank=True)
+
+    # marketing related fields
+    card_image_url = models.URLField(null=True, blank=True)
```
Comment (tuchfarber) [inline course_discovery/apps/journal/models.py:40]: Do we ant to store hero_image_url and basket_image_url? If the hero image is just being used on Journals, then I'd say no to that one. Not sure if ecomm pulls the basket_url from disc or stores it locally.

File: course_discovery/apps/journal/models.py (hunk 1 above)
Comment (bill-filler) [reply course_discovery/apps/journal/models.py]: crap, didn't mean to leave that in. Was getting an error on migration. let me remove it

**Comment by bill-filler** [comment]
@tuchfarber 
updated per comments

**Review by tuchfarber — APPROVED** [review]

**Pull request merged**